### PR TITLE
fix: Update dependabot.yml for auto merge dependencies

### PR DESCRIPTION
**Pull request by jiekechoo**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/master/.github/CONTRIBUTING.md.

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## New Imports
Are there any new imports or modules? If so, what are they used for and why?

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -4,4 +4,15 @@ updates:
...
+    ignore:
+      - dependency-name: "github.com/edgexfoundry/go-mod-core-contracts/v4"
+        # Included when device-sdk-go is updated
+      - dependency-name: "github.com/edgexfoundry/go-mod-messaging/v4"
+        # Included when device-sdk-go is updated
+      - dependency-name: "github.com/edgexfoundry/go-mod-registry/v4"
+        # Included when device-sdk-go is updated
+      - dependency-name: "github.com/edgexfoundry/go-mod-secrets/v4"
+        # Included when device-sdk-go is updated
+      - dependency-name: "github.com/edgexfoundry/go-mod-bootstrap/v4"
```
Comment (cloudxxx8) [inline .github/dependabot.yml:17]:
```suggestion
      - dependency-name: "github.com/edgexfoundry/go-mod-bootstrap/*"
```
use wildcard to save the future version upgrade

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml [hunk 2]
```diff
@@ -14,5 +14,5 @@ updates:
         # Included when device-sdk-go is updated
       - dependency-name: "github.com/edgexfoundry/go-mod-secrets/v4"
         # Included when device-sdk-go is updated
-      - dependency-name: "github.com/edgexfoundry/go-mod-bootstrap/v4"
+      - dependency-name: "github.com/edgexfoundry/go-mod-bootstrap/*"
```
Comment (cloudxxx8) [inline .github/dependabot.yml:17]: please update all the depencencies

Comment (jiekechoo) [reply .github/dependabot.yml:17]: is there any samples? which repo can I refer?

Comment (cloudxxx8) [reply .github/dependabot.yml:17]:
```
    ignore:
      - dependency-name: "github.com/edgexfoundry/go-mod-core-contracts/*"
        # Included when device-sdk-go is updated
      - dependency-name: "github.com/edgexfoundry/go-mod-messaging/*"
        # Included when device-sdk-go is updated
      - dependency-name: "github.com/edgexfoundry/go-mod-registry/*"
        # Included when device-sdk-go is updated
      - dependency-name: "github.com/edgexfoundry/go-mod-secrets/*"
        # Included when device-sdk-go is updated
      - dependency-name: "github.com/edgexfoundry/go-mod-bootstrap/*"
        # Included when device-sdk-go is updated    
```

Comment (jiekechoo) [reply .github/dependabot.yml:17]: got it.

**Pull request closed without merging**